docs/satellites.md

```diff
@@ -0,0 +1,248 @@
+---
+comments: true
+---
+
+# Earth Observation via Satellites
+
+## Definition
+ 
+  Earth Observation (EO) is a discipline that allows us to gather
+  information about our planet. It often involves remote sensing sensors
+  (usually onboard satellites). The data collected using satellites is
+  designated as satellite or geospatial data. Optical (visible, near-
+  and mid-infrared, and thermal infrared) ultraviolet and radar images
+  (microwave) cover the full radiation spectrum beyond the optical
+  domain.
+
+## Setting up satellite imagery
+### Step 1: Which EO Applications Should I Consider?
+
+-   Land (all surface features, such as landscape topography, build-up,
+    soil moisture, vegetation, inland waters, forestry, and agriculture)
+
+-   Ocean (e.g., topography and currents, salinity, and surface
+    temperature)
+
+-   Atmosphere (e.g., aerosols, humidity, cloud particles, ozone, and
+    trace gases)
+
+-   Snow and ice (e.g., sea ice cover and glaciers)
+
+### Step 2: What Information Do I Need?
+
+  Satellites provide an excellent means for tracking changes, that is,
+  comparing baseline data with the actual status and drawing conclusions
+  on the impact data for many sectors and types of projects. Depending
+  on project type, they can also be used for progress monitoring. Since
+  they do not require intermediaries, they are often used for remote
+  verification, but they can be used by all stakeholders for decision making along the project cycle and
+  during every step in the project cycle (e.g., planning and appraisal,
+  implementation, progress monitoring and verification, impact
+  monitoring, and evaluation). Some examples:
+
+-   **Feature mapping** (e.g., buildings, roads, landcover, and water extent)
+
+-   **Change Detection** (e.g., urban growth, deforestation, agricultural expansion, bio-diversity monitoring, climate vulnerability,
+    and glacier melting)
+
+-   **Elevation or Digital Elevation Models** (DEM) (e.g., 3D terrain models for flood risk and 3D urban models)
+
+-   **Monitoring in fragile or disaster contexts** (refugee routes,
+    drought and floods, locusts, etc.)
+
+### Step 3: What Should I Consider When Acquiring Satellite Imagery?
+ 
+  First, define and formulate your information need. Your need (e.g.,
+  detecting deforestation in the Amazon) will direct you to choose the
+  type of satellite image suitable for your case and analysis. Your
+  options:
+
+-   **Open and free-access satellite data:** this includes most of
+    optical and radar satellites from public-funded satellite missions
+    from the EU and the US (such as, Landsat, Sentinel or  ASTER). These data are available with high and low resolutions under
+  free and open data policies and are accessible through several
```

Comment:
please insert spaces from this line (63) until 69 to have the text formatted the same way as the above bold written sections (**open and free-access satelite data:**